plugins/inventory-backend/src/service/standaloneServer.ts

```diff
@@ -16,20 +16,34 @@
 
 import { Server } from 'http';
 import { Logger } from 'winston';
-import { createApplication } from './application';
+import { AggregatorInventory, StaticInventory } from '../inventory';
+import { createStandaloneApplication } from './standaloneApplication';
 
 export interface ServerOptions {
   port: number;
   enableCors: boolean;
   logger: Logger;
 }
 
-export async function startServer(options: ServerOptions): Promise<Server> {
+export async function startStandaloneServer(
```

Comment:
I think this could still be called `startServer`? Isn't it this vs using as middleware?

Reply:
See answer below